hera_pspec/pspecdata.py

```diff
@@ -585,54 +596,105 @@ def C_model(self, key, model='empirical', time_index=None):
             subsequent indices specify the baseline index, in _key2inds format.
 
         model : string, optional
-            Type of covariance model to calculate, if not cached. options=['empirical', 'dsets']
+            Type of covariance model to calculate, if not cached. Options=['empirical', 'dsets', 'autos',...]
             How the covariances of the input data should be estimated.
             in 'dsets' mode, error bars are estimated from user-provided
             per baseline and per channel standard deivations. If 'empirical' is
             set, then error bars are estimated from the data by calculating the
             channel-channel covariance of each baseline over time and
             then applying the appropriate linear transformations to these
-            frequency-domain covariances.
+            frequency-domain covariances. If 'autos' is set, the covariances of the input data
+            over a baseline is estimated from the autocorrelations of the two antennas over channel bandwidth 
+            and integration time. 
 
         time_index : integer, compute covariance at specific time-step in dset
-                       only supported if mode == 'dsets'
+                       supported if mode == 'dsets' or 'autos'
 
+        known_cov : dicts of covariance matrices
+            Covariance matrices that are not calculated internally.
+        
         Returns
         -------
-        C : array-like
+        C : array-like, (Ntimes x spw_Nfreqs x spw_Nfreqs)
```

Comment:
the more standard convention is something like this
`ndarray, (Ntimes, spw_Nfreqs, spw_Nfreqs)`